app/main.development.js

```diff
@@ -38,8 +38,48 @@ const installExtensions = async () => {
   }
 };
 
+const launchDCRd = async () => {
```

Comment:
this function doesn't look async at all. no calls to await.  can this be removed?